### PR TITLE
newJoke/TerraPlana

### PR DESCRIPTION
**Pull request by mariommartins-zz**

### CODE REVIEW

File: jokes.json [hunk 1]
```diff
@@ -22,5 +22,9 @@
   {
     "joke": "Você conhece a piada do fotógrafo? Ainda não foi revelada",
     "author": "Desconhecido"
+  },
+  {
+    "joke": "É claro que a terra é plana! Olha o nome, Planeta! Se fosse redonta era Redondeta!",
```
Comment (danfragoso) [inline jokes.json]: What is "redonta"?

Comment (mariommartins-zz) [reply jokes.json]: It means: "pau no cú dos professores de gramática do repositório"
The issue was revised and corrected

Comment (danfragoso) [reply jokes.json]: What is "cú" with an accented character?

Comment (mariommartins-zz) [reply jokes.json]: it means: "brioco"

Comment (Cardoso222) [reply jokes.json]: 🤣

**Pull request merged**